CHANGELOG.rst

```diff
@@ -7,6 +7,7 @@ Changelog for Mantarray File Manager
 - Added file migration function to go from v0.3.1 to v0.4.1 and v0.4.1 to v0.4.2
 - Bumped H5 File version to 0.4.2
 - Added metadata for original file version from Mantarray, file version prior to migration, and time migration was performed
+- Added method to trim h5 files by indicating the amount of centimilliseconds to be trimmed from the start and end.
```

Comment:
minor note for future: this is a function, not a method.  If a new function/method is important enough to warrant its own changelog entry (which I agree this one is), then it's probably worth including the name of the function in the entry.  If you flank it with backticks, then it looks prettier when the RST gets rendered